pkg/awsutils/awsutils.go

```diff
@@ -277,12 +277,22 @@ type ENIMetadata struct {
 	IPv6Prefixes []*ec2.Ipv6PrefixSpecification
 }
 
+type NetworkCard struct {
+	// max number of interfaces supported per card
+	MaximumNetworkInterfaces int64
+	// the index of current card
+	NetworkCardIndex   int64
+	NetworkPerformance string
```

Comment:
Do we need this somewhere? I see they are populated but not sure where we need use them.